### ISSUE TITLE
Syntax error: word unexpected (expecting ")")

### ISSUE THREAD

**Issue by bananaturtlesandwich** [XAMPPRocky/deploy-mdbook#674]
job ended with failure but there's not enough information for me to know what's wrong
https://github.com/bananaturtlesandwich/UE4-Modding-Guide/actions/runs/5989677648/job/16246137933

**Comment by XAMPPRocky** [comment]
Thank you for your issue! However that seems to be an issue with `mdbook` not `deploy-mdbook`, we just install the latest `mdbook`, if you run your project locally with the latest version you should get the same error.

**Comment by bananaturtlesandwich** [comment]
seems to be fine for me and I'm on latest version :/ 
![](https://github.com/XAMPPRocky/deploy-mdbook/assets/71292624/e2ad869d-a860-4e8a-902b-bb41f0e45823)
![](https://github.com/XAMPPRocky/deploy-mdbook/assets/71292624/84774ad5-528a-46c5-b7c5-d3290767138e)

**Comment by bananaturtlesandwich** [comment]
works from clean too

**Comment by bananaturtlesandwich** [comment]
seems to be a linux issue but I remember the workflow doesn't work on any other platform so can't switch https://stackoverflow.com/questions/36725864/syntax-error-word-unexpected-expecting

**Comment by XAMPPRocky** [comment]
Hmm but there's no usage of `(` in the code? Could you try an older Ubuntu version and see if that works?

**Comment by bananaturtlesandwich** [comment]
there's only one other one - 20.04 and that also fails
https://github.com/bananaturtlesandwich/UE4-Modding-Guide/actions/runs/5989669153/job/16246099138

**Comment by celinval** [comment]
We're having the same issue [here](https://github.com/rust-lang/project-stable-mir/actions/runs/6399194741/job/17370795703). Replacing the action with a manual `cargo install mdbook --locked; mdbook build` succeeds.

**Comment by XAMPPRocky** [comment]
I'm away this week but I think the solution is probably to just update the action's dependencies and then publish a new bundled version, because this doesn't seem like an issue in the action itself.

**Comment by GeckoEidechse** [comment]
bump

**Comment by rbtcollins** [comment]
> I'm away this week but I think the solution is probably to just update the action's dependencies and then publish a new bundled version, because this doesn't seem like an issue in the action itself.

do you need someone to put up a PR ?

**Comment by XAMPPRocky** [comment]
@rbtcollins Yes that would be greatly appreciated, as it's a lot easier to fit code reviews into my schedule than coding time.
